website/src/documentation/programming-guide.md

```diff
@@ -3121,6 +3121,15 @@ public class MyMetricsDoFn extends DoFn<Integer, Integer> {
   }
 }
 ```  
+### 9.5 Export metrics {#export-metrics}
+Beam metrics can be exported to external sinks. If set up the runner will push metrics to a configured sink at a configured period. 
```

Comment:
> If set up the runner will push metrics to a configured sink at a configured period. 

Could you elaborate or rephrase this sentence? I think it misses "then" somewhere since it starts from "if".

Reply:
I asked a native, it is correct but lacks the , after set up. But I'll elaborate a bit to clarify.